README.md

```diff
@@ -144,3 +147,329 @@ Otherwise you will need to tell the VM each time as well like so:
+/path/to/pcov_file2.txt
+/path/to/pcov_file3.txt
+....snip....
+```
+
+###### Outputs
+- merged tsv of proportional coverage 
+```
+CONTIG  START   END     NAME    SAMPLE1    SAMPLE2 SAMPLE3 ....snip....
+1       12191   12227   target1    8.835E-6  1.451E-5     1.221E-5    ....snip....
+1       12596   12721   target2    1.602E-5  1.534E-5     1.318E-5   ....snip....
+....snip....
+```
+
+###### Invocation
+```
+java -Xmx8g -jar  <path_to_hellbender_jar> CombineReadCounts --inputList <text_file_list_of_proportional_coverage_files> \
+    -O <output_merged_file> -MOF 200 
+```
+
+##### Step 3. Create the PoN file
+
+###### Inputs
+- merged tsv of proportional coverage -- generated in step 2.
+
+###### Outputs
+- PoN file -- HDF5 format.  This file can be used for running case samples sequenced with the same process.
+
+###### Invocation
+```
+java -Xmx16g -Djava.library.path=<hdf_jni_native_dir> -jar <path_to_hellbender_jar> CreatePanelOfNormals -I <merged_pcov_file> \
+       -O <output_pon_file_full_path>
+```
+
+
+Running the CNV case and PoN creation Workflows with premade Queue scripts
+--------------------------------------------------------------------------
+
+*These workflows are not officially supported, but are used extensively, internally, for our evaluations.*
+
+#### Requirements
+1. Java 1.8
+2. A functioning GATK4-protected jar (hellbender-protected.jar)
+3. HDF5 1.8.13 
+4. The location of the HDF5-Java JNI Libraries Release 2.9 (2.11 for Macs).  (See platform instructions above for typical locations)  
+5. Queue scala scripts (see instructions below)
+6. Reference genome (fasta) with fai and dict indexes. This can be downloaded as part of the GATK resource bundle: http://www.broadinstitute.org/gatk/guide/article?id=1213
+7. target BED file that was used to create the PoN file.
+8. PoN file (when running case samples only).  This file should be created using the CreatePoNPipeline workflow (example below).
+
+
+#### Download Queue scala scripts
+Click [https://www.broadinstitute.org/gatk/download/gatk4].
+
+Download the following files to the same directory.  This step only needs to be done once.
+- CaseSampleHBExomePipeline.scala
+- CreatePoNPipeline.scala
+- recapseg-hb-eval.jar
+
+#### Test that help can be seen
+```
+java -jar recapseg-hb-eval.jar -S CreatePoNPipeline.scala --help
+```
+
+```
+java -jar recapseg-hb-eval.jar -S CaseSampleHBExomePipeline.scala --help
+```
+
+#### Example case sample run
+
+*Parameters to the Queue jar (recapseg-hb-eval.jar) may need to be adjusted depending on your execution environment*
+
+```
+### Modify parameters below this line
+
+# A simple text file listing each bam file on a separate line
+INPUT_BAMS=case_bam_list.txt
+
+# Output location (absolute paths recommended).  
+#  Seg file will appear in $OUTDIR/tumor_pcov
+#  Calls will appear in $OUTDIR/caller
+OUTDIR=/home/lichtens/evals/out_case/
+
+# Requirement #2 above
+HBJAR=/home/lichtens/hellbender-protected.jar
```

Comment:
recapseg-hb-eval.jar will refer to it as hbjar, if that matters.

Changed it here, though.